docs/source/model-registry.rst

```diff
@@ -78,47 +79,170 @@ version to one of the other valid stages.
+        'name': 'sk-learn-random-forest-reg-model'}
+    ...
+    ...
+
+With hundreds of models, it can be cumbersome to peruse the results returned from this call. A more efficient approach would be to search for a specific model name and list its version
+details using `search_model_versions(...) <https://www.mlflow.org/docs/latest/python_api/mlflow.tracking.html#mlflow.tracking.MlflowClient.search_model_versions>`_ method
+and provide a filter string such as ``“name=’sk-learn-random-forest-reg-model’”``
+
+.. code-block:: py
 
-You can query for the latest version of each registered model by stage using the ``get_latest_versions`` API:
+    client=MlflowClient()
+    [pprint.pprint(dict(mv), indent=4) for mv in client.search_model_versions(“name=’sk-learn-random-forest-reg-model’”)]
+
+    {   'creation_timestamp': 1582671933246,
+        'current_stage': 'Production',
+        'description': 'A random forest model containing 100 decision trees '
+                       'trained in scikit-learn',
+        'last_updated_timestamp': 1582671960712,
+        'name': 'sk-learn-random-forest-reg-model',
+        'run_id': 'ae2cc01346de45f79a44a320aab1797b',
+        'source': './mlruns/0/ae2cc01346de45f79a44a320aab1797b/artifacts/sklearn-model',
+        'status': 'READY',
+        'status_message': None,
+        'user_id': None,
+        'version': 1}
+
+    {   'creation_timestamp': 1582671960628,
+        'current_stage': 'None',
+        'description': None,
+        'last_updated_timestamp': 1582671960628,
+        'name': 'sk-learn-random-forest-reg-model',
+        'run_id': 'd994f18d09c64c148e62a785052e6723',
+        'source': './mlruns/0/d994f18d09c64c148e62a785052e6723/artifacts/sklearn-model',
+        'status': 'READY',
+        'status_message': None,
+        'user_id': None,
+        'version': 2
+    }
+
+Deleting Registered Models
+--------------------------
+You can either delete a specific version of the registered model or registered model and all its versions.
+This operation is irrevocable, so use it judiciously.
 
 .. code-block:: py
 
-   client.get_latest_versions(name = "EmailMarketingCampaign",
-                              stages = ["Staging"])
+    client=MlflowClient()
+    # Delete a specific version of the model
+    client.delete_model_version(name=”sk-learn-random-forest-reg-model”,
+            version=1 )
+    # Delete the registered model along with all its versions
+    client.delete_registered_model(name=”sk-learn-random-forest-reg-model”)
 
-Newly-registered model versions are added to the registry with stage **None**. To update the stage, call the ``update_model_version`` API:
+However, if you are unsure, you may want to archive specific versions of a registered model.
```

Comment:
Maybe flip these. We actually expect people to archive most often. Maybe the session could be "Archiving and deleting registered models" and we first say how to archive and then say if you want to permanently delete, use this instead.

Reply:
Makes sense, better to first Archive them, and later they can always delete once they are certain.